Opensourcecommunity/Governance.md

```diff
@@ -44,3 +42,5 @@ Current maintainers:
 * Open smart grid platform and smart lighting domain: [Kevin Smeets](https://github.com/kevinsmeets)
 * Smart metering domain: [Bart van der Zwet](https://github.com/bvdzwet)
 * Documentation & configutation: [Ruud Lemmers](https://github.com/rlemmers)
```

Comment:
Please change:  Documentation & configutation: [Ruud Lemmers](https://github.com/rlemmers)
into 
Documentation & configuration: [Ruud Lemmers](https://github.com/rlemmers)

Then close https://github.com/OSGP/Documentation/pull/117

This prevents a merge conlfict, and since its a small change i suggest we fix the spelling correction in this PR and close PR-117